docs/product-release-notes/src/main/asciidoc/con-bxms-rn-introduction.adoc

```diff
@@ -9,13 +9,18 @@ ifdef::BPMS[]
 {PRODUCT} uses a centralized repository where all resources are stored. This ensures consistency, transparency, and the ability to audit across the business. Business users can modify business logic and business processes without requiring assistance from IT personnel.
 
 To facilitate the Business Rules component, Red Hat JBoss BPM Suite includes integrated Red Hat JBoss BRMS.
+
+{PRODUCT} {PRODUCT_VERSION} {Release} with integrated Red Hat JBoss BRMS further expands the product capabilities by providing an enhanced user experience, a simplified process design workflow, case management capabilities, an improved comprehensive dashboard, and optimized task handling and collaboration features.
+
 endif::BPMS[]
 
 ifdef::BRMS[]
 Red Hat JBoss BRMS is an open source decision management platform that combines Business Rules Management and Complex Event Processing. It automates business decisions and makes that logic available to the entire business.
 
 Red Hat JBoss BRMS uses a centralized repository where all resources are stored. This ensures consistency, transparency, and the ability to audit across the business. Business users can modify business logic without requiring assistance from IT personnel.
 
+{PRODUCT} {PRODUCT_VERSION} {Release} with integrated Red Hat JBoss BRMS further expands the product capabilities by providing an enhanced user experience, a simplified process design workflow, case management capabilities, an improved comprehensive dashboard, and optimized task handling and collaboration features.
```

Comment:
This doesn't seem to make sense to add this to the BRMS section?  Probably doesn't matter too much now as for LA we only build BPMS but might want to improve this for the future.